addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -37,30 +37,45 @@ def _get_wizard_values(self):
     # COMPUTE/CONSTRAINS METHODS
     # -------------------------------------------------------------------------
 
+    @api.depends('l10n_it_edi_actionable_errors')
+    def _compute_l10n_it_edi_warning_message(self):
+        # To be removed -- Proxy feature to be replaced with actionable_errors as soon as the user updates the module
+        for wizard in self:
+            messages = []
+            if wizard.l10n_it_edi_actionable_errors:
+                messages.append(escape(_("Please upgrade the Italian EDI module to update this widget.")))
+                messages.append(escape(_("Go to Applications page and update the 'Italia - Fatturazione Elettronica' module.")))
+                messages.append(Markup())
+                for error_key, error_data in wizard.l10n_it_edi_actionable_errors.items():
+                    message = error_data['message']
+                    split = error_key.split("_")
+                    if len(split) > 1 and (model_id := {'partner': 'res.partner', 'move': 'account.move', 'company': 'res.company'}.get(split[0], None)):
+                        records = self.env[model_id].browse(error_data['action_params'][0])
+                        message = f"{message} - {', '.join(records.mapped('display_name'))}"
+                    messages.append(Markup(escape(message)))
+                wizard.l10n_it_edi_warning_message = nl2br(Markup("\n").join(messages))
```

Comment:
You don't need all these escape if you join with a markup
```
>>> Markup("\n").join([Markup("<abc>"), "<def>"])
Markup('<abc>\n&lt;def&gt;')
```

Reply:
and then it's pretty weird to join on `Markup("\n")`, are you sure you didn't mean `Markup("<br/>")`?

Reply:
What you say is fine, yet since there already is an `nl2br` outside for the content of the warnings, so I think it will only save a single replace.

Reply:
Sure! Ok.